app/presenters/related_items.rb

```diff
@@ -0,0 +1,37 @@
+class RelatedItems
+  attr_reader :contact
+
+  def initialize(contact)
+    @contact = contact
+  end
+
+  def payload
+    { sections: build_sections }
+  end
+
+private
+
+  def build_sections
+    sections = []
+
+    if contact.quick_links.to_a.any?
```

Comment:
Turns out that `quick_links` isn't actually required in the schemas, so it could potentially be nil:

https://github.com/alphagov/govuk-content-schemas/blob/1713643feeb4931330cfdd5862cebbe1d525d4af/formats/contact/publisher/details.json#L5-L7